### ISSUE TITLE
Images Won't Load

### ISSUE THREAD

**Issue by liocalset** [wireapp/wire-webapp#689]
Hi,

There is a new issue I am experiencing with the wire web app as of one of the recent updates. When I receive or send an image, all that's displayed is a frame that's the same dimensions as the image. The frame displays as a turquoise, beige, or greenish box (when I tested, the color varied between sessions) with three animated dots in the center indicating the image is loading. However, the image never loads. The web app continues trying to unsuccessfully load the image indefinitely.

Viewing the FF browser console, here are the errors I just got when sending a picture to a friend:

`Failed to load ‘https://prod-nginz-https.wire.com/conversations/a81e40f5-63c8-4b21-a1ef-6585af0dd526/otr/assets/bdb2c02d-8853-487b-8b24-c44225c702f2?access_token=o81CQHLqkMT_PurV9n8HUwflR1eMAMXPe3NUHyy6aoFfXBa2EXraElkU8_vlkogzGTw2WAusBhxv6brTl36bBA==.v=1.k=1.d=1485531707.t=a.l=.u=3c2e2553-6e13-4f65-9fb1-4358c83e6d93.c=6531907568139570110&forceCaching=true’. A ServiceWorker passed a promise to FetchEvent.respondWith() that rejected with ‘TypeError: NetworkError when attempting to fetch resource.’.`

`Failed to load ‘https://prod-nginz-https.wire.com/assets/a20bc881-cdad-4f3f-b222-4fb71c9e68d1?access_token=o81CQHLqkMT_PurV9n8HUwflR1eMAMXPe3NUHyy6aoFfXBa2EXraElkU8_vlkogzGTw2WAusBhxv6brTl36bBA==.v=1.k=1.d=1485531707.t=a.l=.u=3c2e2553-6e13-4f65-9fb1-4358c83e6d93.c=6531907568139570110&conv_id=6bf01ba2-f5b5-4bbc-823e-222de6cf6793&forceCaching=true’. A ServiceWorker passed a promise to FetchEvent.respondWith() that rejected with ‘TypeError: NetworkError when attempting to fetch resource.’.`

System Specs:

OpenSUSE Leap 42.2
FireFox 50.1.0
wire-webapp Version 2017-01-27

Thanks!

**Comment by MakisH** [comment]
For me, this works normally on Firefox (Firefox 50.1.0, Ubuntu 16.04, same wire version).

**But**, I had the same problem when using Wire via Rambox 0.4.5 (Rambox is an Electron app). It works correctly on Rambox 0.5.3.

**Comment by liocalset** [comment]
I did some more testing and narrowed down this issue to [Privacy Badger](https://addons.mozilla.org/en-US/firefox/addon/privacy-badger17/) 2017.1.26.1. This add-on has to be completely disabled for images to load in the wire web app, even though Privacy Badger isn't blocking any trackers as there are none to block. I also tried setting the option in Privacy Badger: "Disable Privacy Badger for this Site", but had the same problem with images not loading.
